### ISSUE TITLE
When is the next official release of impacket having support for python 3.8 ? 

### ISSUE THREAD

**Issue by samruddhi2909** [SecureAuthCorp/impacket#952]
Tentative dates will help.

**Comment by asolino** [comment]
Hey @samruddhi2909, targeting mid Q4 for the next release.

**Comment by rhertzog** [comment]
> Hey @samruddhi2909, targeting mid Q4 for the next release.

Mid-Q4 should be this week if my count is correct ;-)

Speaking as a Kali developer, it would be really nice to have a release soon as we get more and more reports of people having issues because Debian (and thus Kali) switched to Python 3.8 and Debian waits on upstream releases so they still have impacket 0.9.21 without all the Python 3.8 fixes.

**Comment by asolino** [comment]
Hey @rhertzog 

You can do math! :). Indeed we're working as I write closing and testing as much as possible issues and PRs and preparing collaterals. We're targeting having everything ready by the week of the 23rd (before TG).

> Speaking as a Kali developer, it would be really nice to have a release soon as we get more and more reports of people having issues because Debian (and thus Kali) switched to Python 3.8 and Debian waits on upstream releases so they still have impacket 0.9.21 without all the Python 3.8 fixes.

Couldn't agree with you more, I also saw a lot of Kali users as well asking about newer impacket versions. It's great to see the amount of people in the community interested in impacket's features :).

**Comment by asolino** [comment]
Fixed at https://github.com/SecureAuthCorp/impacket/releases/tag/impacket_0_9_22 . Closing.
